### PR TITLE
Create About Section-Single page layout

### LINKED ISSUE #80: [Site] About Section-Single page layout
**Current State of the GetNighthawk.dev site**
Empty about page

**Desired State of the GetNighthawk.dev site**
Requires a single page layout/design for the about section.
@austinogiza 

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by Anita-ihuman**
**Description**
Created a single page layout for About section
This PR fixes #80 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vinayaksh42** [comment]
@Anita-ihuman the font used on the `about us` page seems to be different from the `home` page
About us page:
chrome_fhby0xuMzy

Home page:
chrome_WbdNRlLULB

**Comment by Anita-ihuman** [comment]
I am working on the font right now

File: docs/_sass/about.scss [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+.hero-section-smp {
+  width: 85%;
+  height: 100%;
+}
+.whyGetnighthawk-text {
+  font-size: 20px;
+  text-align: left;
+}
+.hero-section_content .text {
```
Comment (vinayaksh42) [inline docs/_sass/about.scss:15]: a come "," is required if you want to add the same style to multiple classes
```css
.hero-section_content, .text {
```

File: docs/_sass/about.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  font-family: Open Sans;
+}
+.whyGetnighthawk {
+  justify-content: space-around;
+  display: flex;
+  padding-top: 40px;
+  padding-left: 30px;
+}
+
+.whyGetnighthawk .text {
```
Comment (vinayaksh42) [inline docs/_sass/about.scss:29]: a come "," is required if you want to add the same style to multiple classes
```css
.whyGetnighthawk , .text {
```

File: docs/_sass/about.scss [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+.nighthawk-meshery-fig {
+  display: flex;
+  justify-content: center;
+  padding-top: 40px;
+}
+.figbox {
+  margin: 0px 40px;
+  padding: 20px 0px;
+}
+.nighthawk-meshery-fig img {
```
Comment (vinayaksh42) [inline docs/_sass/about.scss:44]: same applies over here

File: docs/_sass/about.scss [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  font-style: normal;
+  font-weight: 600;
+  font-size: 34px;
+  line-height: 34px;
+  color: #000000;
+  overflow-wrap: normal;
+}
+
+.hero-section_bold {
+  font-family: Qanelas Soft;
```
Comment (vinayaksh42) [inline docs/_sass/about.scss:64]: Kindly change the font to that of `home` page

File: docs/_sass/about.scss [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+}
+.nighthawk-meshery-fig img {
+  padding: 0px 30px;
+  object-fit: cover;
+  width: 450px;
+  height: 100%;
+  border-radius: 20px;
+}
+
+.whyGetnighthawk-bold h4 {
```
Comment (vinayaksh42) [inline docs/_sass/about.scss:52]: same applies over here

**Comment by vinayaksh42** [comment]
You are also required to sign your commits, kindly refer to this https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#contributor-instructions if you are facing an issue regarding signing commits. Also, it is advised if you push more commits against the same PR so that all the discussion done under it remains available with respect to the issue.

**Pull request closed without merging**